<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActivityDialog</class>
 <widget class="QDialog" name="ActivityDialog">
  <property name="windowTitle">
   <string>Add earlier activity</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabTimed">
      <attribute name="title">
       <string>Timed</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="lblActivity">
         <property name="text">
          <string>Activity:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QDateEdit" name="deStartedAt">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTimeEdit" name="teStartedAt">
           <property name="currentSection">
            <enum>QDateTimeEdit::HourSection</enum>
           </property>
           <property name="displayFormat">
            <string>HH:mm</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblTo">
           <property name="text">
            <string>to</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTimeEdit" name="teEndedAt">
           <property name="currentSection">
            <enum>QDateTimeEdit::HourSection</enum>
           </property>
           <property name="displayFormat">
            <string>HH:mm</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="deEndedAt">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbInProgress">
           <property name="text">
            <string>in progess</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <widget class="TagsLineEdit" name="leTags"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblTags">
         <property name="text">
          <string>Tags:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="ActivityLineEdit" name="leActivity">
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblTime">
         <property name="text">
          <string>Time:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabUntimed">
      <attribute name="title">
       <string>Untimed</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QLabel" name="lblUntimedDuration">
         <property name="text">
          <string>Duration:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="ActivityLineEdit" name="leUntimedActivity"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblUntimedTags">
         <property name="text">
          <string>Tags:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblUntimedActivity">
         <property name="text">
          <string>Activity:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QDateEdit" name="deUntimedDate">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QSpinBox" name="sbUntimedHours">
           <property name="maximum">
            <number>23</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblUntimedHours">
           <property name="text">
            <string>hours</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sbUntimedMinutes">
           <property name="maximum">
            <number>59</number>
           </property>
           <property name="singleStep">
            <number>5</number>
           </property>
           <property name="value">
            <number>15</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblUntimedMinutes">
           <property name="text">
            <string>minutes</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblUntimedDate">
         <property name="text">
          <string>Date:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="TagsLineEdit" name="leUntimedTags"/>
       </item>
       <item row="4" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
      <zorder>lblUntimedDuration</zorder>
      <zorder>lblUntimedDate</zorder>
      <zorder>lblUntimedActivity</zorder>
      <zorder>leUntimedActivity</zorder>
      <zorder>lblUntimedTags</zorder>
      <zorder>leUntimedTags</zorder>
      <zorder>verticalSpacer_2</zorder>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ActivityLineEdit</class>
   <extends>QLineEdit</extends>
   <header>activitylineedit.h</header>
  </customwidget>
  <customwidget>
   <class>TagsLineEdit</class>
   <extends>QLineEdit</extends>
   <header>tagslineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>deStartedAt</tabstop>
  <tabstop>teStartedAt</tabstop>
  <tabstop>teEndedAt</tabstop>
  <tabstop>deEndedAt</tabstop>
  <tabstop>cbInProgress</tabstop>
  <tabstop>leActivity</tabstop>
  <tabstop>leTags</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>deUntimedDate</tabstop>
  <tabstop>sbUntimedHours</tabstop>
  <tabstop>sbUntimedMinutes</tabstop>
  <tabstop>leUntimedActivity</tabstop>
  <tabstop>leUntimedTags</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ActivityDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ActivityDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leActivity</sender>
   <signal>textEdited(QString)</signal>
   <receiver>leUntimedActivity</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leUntimedActivity</sender>
   <signal>textEdited(QString)</signal>
   <receiver>leActivity</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leTags</sender>
   <signal>textEdited(QString)</signal>
   <receiver>leUntimedTags</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leUntimedTags</sender>
   <signal>textEdited(QString)</signal>
   <receiver>leTags</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>401</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
